### ISSUE TITLE
Where can I find examples of using this boilerplate?

### ISSUE THREAD

**Issue by dwschulze** [bencodezen/vue-enterprise-boilerplate#221]
Are there examples of correct use of idioms and patterns for this boilerplate?  Something more than a Hello World example if any are available.

Thanks.

**Comment by M-Abdullahi** [comment]
We are using it for a hospital management system but the repository is private. 
Ironically, I've discovered this project from a theme that we  purchased from themeforest.
The code structure and file organization of this project was kinda sleek to me back then. I ended up spending 3 months learning 
how everything worked.


![Screenshot from 2021-08-13 22-06-29](https://user-images.githubusercontent.com/14330689/129407545-09ad7e2d-6312-4361-aa69-d47e956f9e44.png)

![config](https://user-images.githubusercontent.com/14330689/129407579-7c684995-b459-42c6-9b83-fe42589a19d1.png)
